dkarlovi/symfony-flex-docker/0.1/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+    "copy-from-package": {
+        ".infra/": ".infra",
+        ".dockerignore": ".dockerignore",
+        "docker-compose.yaml": "docker-compose.yaml"
+    },
+    "env": {
+        "COMPOSE_PROJECT_NAME": "symfony",
```

Comment:
Agree, but this is a placeholder.

I'd like to get some information from `composer.json` (such as project name), hope Flex will provide access to it while copying files. /cc @fabpot